lib/rrule/weekday.rb

```diff
@@ -4,6 +4,16 @@ module RRule
   class Weekday
     attr_reader :index, :ordinal
 
+    DAY_NAMES = %w[
```

Comment:
Possible to move these new methods to the `Humanizer` class? I think it'd be nice to have all the code that deals with humanizing rules in one place.